dotcom-rendering/src/web/lib/layoutHelpers.ts

```diff
@@ -25,6 +25,15 @@ export const decideLineCount = (design?: ArticleDesign): 8 | 4 => {
 	return 4;
 };
 
+export const decideLineColour = (
+	design?: ArticleDesign,
+): string | undefined => {
+	if (design === ArticleDesign.LiveBlog) {
+		return 'rgba(255, 255, 255, 0.4)';
+	}
+	return undefined;
+};
+
```

Comment:
We can inline this abstraction. It's not used anywhere else and it's good to keep code as close to possible to where it is being used so that people reading it in future don't have to open multiple files to see what is happening unless they really need to

Reply:
I was also doubtful about this. Made it inline within where it was needed